.github/workflows/push-docker-image.yml

```diff
@@ -0,0 +1,46 @@
+name: Push docker image to harbor
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches: [ main ]
+
+jobs:
+  test-and-push-image:
```

Comment:
This should happen before the release and not during the release. Take a look at [the Delivery Pipeline Concept](https://github.com/puzzle/delivery-pipeline-concept).